subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/OutputScrapingExecutionResult.java

```diff
@@ -166,9 +166,9 @@ private String normalize(LogContent output) {
             } else if (line.contains(DaemonStateCoordinator.DAEMON_WILL_STOP_MESSAGE)) {
                 // Remove the "Daemon will be shut down" message
                 i++;
-            } else if (line.contains(LowHeapSpaceDaemonExpirationStrategy.EXPIRE_DAEMON_MESSAGE)) {
-                // Remove the "Expiring Daemon" message
-                i++;
+            } else if (line.contains(HealthExpirationStrategy.EXPIRE_DAEMON_MESSAGE)) {
+                // Remove the "The Daemon will expire" message
+                i+=6;
```

Comment:
Not sure how I'd go about doing that. Since we're reading the output line by line directly from the builds output we don't have access to the original message. What were you thinking here more concretely?

Reply:
I wasn't considering how difficult this would be to do, just highlighting it.  If it's not worth it, no big deal.

Reply:
Yeah I'm going to leave this one as is